src/python/pants/reporting/zipkin_reporter.py

```diff
@@ -176,5 +184,8 @@ def bulk_record_workunits(self, engine_workunits):
         flags='0', # flags: stores flags header. Currently unused
         is_sampled=True,
       )
-      span.start_timestamp = workunit['start_timestamp']
+      span.start_timestamp = start_timestamp
       span.stop()
+
+def from_secs_and_nanos_to_float(secs, nanos):
+  return secs + nanos/NUM_NANOSECS_IN_SEC
```

Comment:
feel free to ignore, but some parens around the division would make me feel good.